ReactiveXComponent/Connection/IXCSession.cs

```diff
@@ -6,7 +6,9 @@ namespace ReactiveXComponent.Connection
     public interface IXCSession : IDisposable
     {
         bool IsOpen { get; }
+        event EventHandler SessionOpened;
         event EventHandler SessionClosed;
+        event EventHandler<System.IO.ErrorEventArgs> ConnectionError;
```

Comment:
On the JS API we called it DisconnectionError because what we are handling here is the case of a disconnection after connection.

Reply:
This event is raised when we encounter a problem while connecting. That's why it's called ConnectionError. You can refer to _WebSocketClient.Init()_ method.